### PR TITLE
Update prepare-tls-1.2-in-office-365.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the link for supported cipher suites as the old FAW page does not have that information any more.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7649

File: microsoft-365/compliance/prepare-tls-1.2-in-office-365.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The [Microsoft TLS 1.0 implementation](https://support.microsoft.com/help/311733
 
 For information about how to remove TLS 1.0 and 1.1 dependencies, see the following white paper: [Solving the TLS 1.0 problem](https://www.microsoft.com/download/details.aspx?id=55266).
 
-After you upgrade to TLS 1.2, make sure that the cipher suites you're using are supported by Azure Front Door. Microsoft 365 and Azure Front Door have slight differences in cipher suite support. For details, see [What are the current cipher suites supported by Azure Front Door?](/azure/frontdoor/front-door-faq#what-are-the-current-cipher-suites-supported-by-azure-front-door-).
+After you upgrade to TLS 1.2, make sure that the cipher suites you're using are supported by Azure Front Door. Microsoft 365 and Azure Front Door have slight differences in cipher suite support. For details, see [What are the current cipher suites supported by Azure Front Door?](/azure/frontdoor/concept-end-to-end-tls#supported-cipher-suites)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prepare-tls-1.2-in-office-365.md]:
```suggestion
After you upgrade to TLS 1.2, make sure that the cipher suites you're using are supported by Azure Front Door. Microsoft 365 and Azure Front Door have slight differences in cipher suite support. For details, see [What are the current cipher suites supported by Azure Front Door?](/azure/frontdoor/concept-end-to-end-tls#supported-cipher-suites).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder

**Comment by pshelton-skype** [comment]
@JohanFreelancer9 or @mjcaparas, can you please merge?

**Comment by yogkumgit** [comment]
@KCCross Kindly merge this PR. Thanks

**Pull request merged**